package NYT.NObjectClient.NProto;

////////////////////////////////////////////////////////////////////////////////

message TReqExecute
{
    // Each Execute call carries a number of YPath requests.
    // Each of these requests is represented by a message.
    // Each message, in turn, is a sequence of parts.
    // This array contains the lengths of those part sequences.
    // In particular, its length is just the number of encaplusated YPath requests.
    repeated int32 part_counts = 1;
}

message TRspExecute
{
    // Similar to TReqExecute:::part_counts, but denotes the number of parts
    // in each encapsulated response.
    repeated int32 part_counts = 1;
}

////////////////////////////////////////////////////////////////////////////////

message TReqGCCollect
{ }

message TRspGCCollect
{ }

////////////////////////////////////////////////////////////////////////////////
